### PR TITLE
Updated template part for French market

### PR DESCRIPTION
**Pull request by magdignomad**

### CODE REVIEW

File: doc/handbook-fr/configuration.md [hunk 1]
```diff
@@ -478,4 +481,330 @@ As last parameter the endpoint for the ChaîneCloude product has to be defined:
...
+
+For this purpose, fiskaltrust provides an HTTP API which can be used in the sandbox (https://helipad-sandbox.fiskaltrust.cloud/api/configuration) but also in the live environment (https://helipad.fiskaltrust.cloud/api/configuration).
+
+#### API-Call
+
+Various requirements are necessary to call up the API.
+
+1. You need the `AccountID` and the `AccessToken`, which you can find in your company account (https://portal-sandbox.fiskaltrust.fr/AccountProfile) at the bottom of the page.
+2. You need the _SIRET_, _Company Name_ and _Outlet_-Number.
+3. The template as JSON strong.
```
Comment (AxelKutschera) [inline doc/handbook-fr/configuration.md]:
```suggestion
3. The template as JSON string.
```

**Review by AxelKutschera — DISMISSED** [review]
one typo 
- test with postman not working (CashBoxIdentification = ftCashBoxIdentification)

**Review by magdignomad — COMMENTED** [review]
Changed links

File: doc/handbook-fr/configuration.md [hunk 2]
```diff
@@ -434,24 +434,27 @@ To initialize a SCU for a queue the following block can be used:
...
-**Example of a standard template:**
+##### Example of a standard template
 
 ```
 {
     "ftCashBoxId": "|[cashbox_id]|",
     "ftQueues": [{
             "Id": "|[queue0_id]|",
-            "Package": "fiskaltrust.service.azure",
+            "Package": "fiskaltrust.Middleware.azure",
```
Comment (StefanKert) [inline doc/handbook-fr/configuration.md]: This is wrong :) should still be service

Comment (StefanKert) [reply doc/handbook-fr/configuration.md:457]: Also the signaturcloud can´t be produced through a template.

File: doc/handbook-fr/configuration.md [hunk 3]
```diff
@@ -387,7 +387,7 @@ A template always start with
      "ftQueues": [​{​"Id": "|[queue0_id]|",
 ```
 
-Then add queue type: `"Package": "fiskaltrust.service.azure",`
+Then add queue type: `"Package": "fiskaltrust.Middleware.azure",`
```
Comment (StefanKert) [inline doc/handbook-fr/configuration.md]: Same here :)

File: doc/handbook-fr/configuration.md [hunk 4]
```diff
@@ -434,24 +434,27 @@ To initialize a SCU for a queue the following block can be used:
 ```
 "init_ftSignaturCreationUnitFR": [{
         "ftSignaturCreationUnitFRId": "|[scu0_id]|",
-        "Siret": "|[siret]|"
+        "Siret": "|[siret]|",
+        "CompanyName": "|[company_name]|",
+        "CountryCode": "FR",
+        "OutletNumber": "|[outlet_number]|"
     }
```
Comment (StefanKert) [inline doc/handbook-fr/configuration.md:441]: Do you want to add a closing ] here?

Comment (magdignomad) [reply doc/handbook-fr/configuration.md:441]: added

**Pull request merged**